src/CheckCurrentStableVersionCommand.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Phphub;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\DomCrawler\Crawler;
+
+final class CheckCurrentStableVersionCommand extends Command
+{
+    private const DOWNLOADS_PAGE = 'https://www.php.net/downloads.php';
+    private const MESSAGE = 'Current stable PHP version is %s';
+
+    protected function configure(): void
+    {
+        $this->setName('phphub:check-current-stable-version')
```

Comment:
```suggestion
$this->setName('last-stable-version');
```
Just to avoid the phphub inclusion everytime.
WDUT?

Reply:
Why not 😃